front-end/src/components/SideNavigation.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
+import {
+    Drawer,
+    Box,
+    IconButton,
+    List,
+    ListItem,
+    ListItemButton,
+    ListItemIcon,
+    ListItemText,
+    Divider,
+    Tooltip
+} from '@mui/material';
+//Icons
+import MenuIcon from '@mui/icons-material/Menu';
+import DashboardIcon from '@mui/icons-material/DashBoard'
```

Comment:
Rollup is unable to resolve this import `@mui/icons-material/DashBoard` thus build is failing. Please verify this is being imported correctly.

Reply:
It was because it was "Dashboard" with a lowercase b instead of uppercase, I fixed it. I also removed unused imports. I had a bunch of them because I originally was planning on having everything in one component but ended up splitting things up into a main component and helper component, but imported the same on both. I know unused imports could be a problem later so I removed them.

Reply:
thanks!